services/processor/native/service.go

```diff
@@ -7,26 +7,22 @@ import (
 	"github.com/orbs-network/orbs-spec/types/go/protocol"
 	"github.com/orbs-network/orbs-spec/types/go/services"
 	"github.com/orbs-network/orbs-spec/types/go/services/handlers"
+	"github.com/pkg/errors"
 )
 
 type service struct {
```

Comment:
why the name `service`? it's an overloaded name as it is, now you're adding more files / entities with that name. What's wrong with `processor`?

Reply:
It's a style question. Eventually, `service.go` files are just routing/boilerplate that hook up your actual service implementation to the interfaces that the spec dictates

Because it's only a routing file with almost zero logic, I don't want to give it a domain meaningful name